src/tiledb/cloud/vcf/ingestion.py

```diff
@@ -715,6 +715,7 @@ def file_size(uri: str) -> int:
                         if records is None:
                             status = "" if status == "ok" else status + ","
                             status += "bad index"
+                            records = 0
```

Comment:
@alancleary I may be missing something, but is this working around a defect of `get_record_count()`? Should that function be returning `0` instead of `None`?

Reply:
The return description of `get_record_count()` is "record count or None if there is an error". So `None` is expected since the bad index prevents the function from running correctly. I think that makes sense because there's clear a distinction between valid outputs and errors, which is used to set the manifest status appropriately. That said, I can't imagine a user will ever have data where the record count is 0!

Reply:
I get it.